tcms/rpc/api/forms/testrun.py

```diff
@@ -21,28 +23,17 @@ def assign_plan(self, plan_id):
         self.populate(self.fields['plan'].queryset.first().product_id)
 
 
-class UpdateForm(NewForm):
-    plan = forms.ModelChoiceField(
-        queryset=TestPlan.objects.all(),
+class UpdateForm(forms.ModelForm):
+
+    manager = UserField()
+    default_tester = UserField(required=False)
+
+    start_date = forms.DateTimeField(
         required=False,
```

Comment:
These shouldn't be here. The loop in the constructor does this for you and the whole idea of migrating to ModelForm is to not have explicitly define fields.

Reply:
I get the idea about ModelForm, but in this case we need to define the field, because of the additional format information, which is not present in `TestRun::start_date` and `TestRun::stop_date`

Reply:
Your format is definitely inappropriate b/c it loses precision. Also it differs from the default behavior which means values going back and forth are not the same.

See MODERNRPC_JSON_DECODER and MODERNRPC_JSON_ENCODER and the implementation of DjangoJSONEncoder. datetime serialization to string uses .isoformat(). 

And there is no corresponding json decoder to be used.